vic/drivers/image/src/vic_force.c

```diff
@@ -59,6 +59,9 @@ vic_force(void)
     size_t                     d4count[4];
     size_t                     d4start[4];
 
+    cv_converter              *converter;
```

Comment:
is this basically a double?  if so, we may be able to put this in `param_set_struct` so we only have to check the units and generate the converter once.